src/components/FormComponents/CheckBox/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const CheckBox = (props) => {
+  return (
+    <input type="checkbox" {...props} />
```

Comment:
Although using the spread operator is really useful in some cases, and it really shortens the code, you need to care about what are the valid DOM attributes when using it on html elements (Just a note/reminder - in the code here you're using everything properly)